supplywise-ui/src/pages/management/restaurants.js

```diff
@@ -59,13 +49,40 @@ export default function Restaurants() {
                 console.log(message);
             })
             .catch((error) => console.error(error));
-    sessionStorage.removeItem('loggedUser');
     window.location.reload();
 };
 
 
   return (
     <DashboardLayout>
+      <div className="row">
+        <div className="col"></div>
+        <div className="col-auto">
+          <button type="button" className="btn btn-primary" data-bs-toggle="modal" data-bs-target="#createRestaurantModal">Add Restaurant</button>
```

Comment:
The button color should align with the existing system design, matching the "sw-bgcolor" class used for other buttons. Can you fix this?

Reply:
I will resolve these changes in the next branch.